client/Helpers/UI.elm

```diff
@@ -241,17 +242,17 @@ starIcon msgAdd msgRemove store id =
             [ text starOff ]
 
 
+appNameToAppId : String -> String
+appNameToAppId name =
+    if name |> String.startsWith appPreffix then
+        String.dropLeft 6 name
```

Comment:
What is 6? The size of `stups_`? In this case maybe it should not be a magic value, especially since you moved `stups_` to appPrefix in Config

Reply:
Wow! You are really reading all this! Good catch! Thank you!